### ISSUE TITLE
Ratio of training set to testing set

### ISSUE THREAD

**Issue by AmosJoseph** [brendenpetersen/deep-symbolic-optimization#81]
Hi, I notice the training parameter like below：
   
 "training": {
      "n_samples": 2000000,
      "batch_size": 1000,
      "epsilon": 0.05,
      "baseline": "R_e",
      "alpha": 0.5,
      "b_jumpstart": false,
      "n_cores_batch": 2,
      "complexity": "token",
      "const_optimizer": "scipy",
      "const_params": {
         "method": "L-BFGS-B",
         "options": {
            "gtol": 0.001
         }
      },

What‘s the training set size and the testing set size?
 And what's ratio of training set to testing set?
 
 Best!

**Comment by brendenpetersen** [comment]
DSO is a generic optimization framework. The notion of train/test set, in general, doesn't really apply.

If you are looking at the `regression` task specifically, the train/test splits are specified for each benchmark in [dso/task/regression/benchmarks.csv](https://github.com/brendenpetersen/deep-symbolic-optimization/blob/master/dso/dso/task/regression/benchmarks.csv).
